### PR TITLE
Fix unittests and etcd paths

### PR DESCRIPTION
**Pull request by Denchick**

### CODE REVIEW

File: router/server/server.go [hunk 1]
```diff
@@ -24,6 +24,7 @@ type Server interface {
 	AddDataShard(clid string, shardKey kr.ShardKey, tsa string) error
 	UnRouteShard(sh kr.ShardKey, rule *config.FrontendRule) error
 	Datashards() []shard.Shard
+	DatashardIds() []string
```
Comment (reshke) [inline router/server/server.go:28]: Why do we need another method in server iface? We can write separate utility function which accepts []shard.Shard and returns []string

**Review by reshke — CHANGES_REQUESTED** [review]

File: router/server/shard.go [hunk 2]
```diff
@@ -144,7 +144,7 @@ func (srv *ShardServer) Receive() (pgproto3.BackendMessage, error) {
 		Uint("single-shard", spqrlog.GetPointer(srv)).
 		Type("message-type", msg).
 		Str("txstatus", srv.TxStatus().String()).
-		Str("shard", srv.Datashards()[0].ID()).
+		Strs("shards", srv.DatashardIds()).
```
Comment (EinKrebs) [inline router/server/shard.go:147]:
```suggestion
		Strs("shards", srv.DatashardIds()[0]).
```
Зачем убирать индекс?

File: router/relay/relay.go [hunk 3]
```diff
@@ -608,7 +608,7 @@ func (rst *RelayStateImpl) ProcQuery(query pgproto3.FrontendMessage, waitForResp
 	}
 
 	spqrlog.Zero.Debug().
-		Str("server", server.Datashards()[0].ID()).
+		Strs("shards", server.DatashardIds()).
```
Comment (EinKrebs) [inline router/relay/relay.go:611]:
```suggestion
		Strs("shards", server.DatashardIds()[0]).
```

File: router/relay/relay.go [hunk 4]
```diff
@@ -918,7 +918,7 @@ func (rst *RelayStateImpl) DeployPrepStmt(qname string) (shard.PreparedStatement
 		Str("query", query).
 		Uint64("hash", hash).
 		Str("client", rst.Client().ID()).
-		Str("shard", rst.Client().Server().Datashards()[0].ID()).
+		Strs("shards", rst.Client().Server().DatashardIds()).
```
Comment (EinKrebs) [inline router/relay/relay.go:921]:
```suggestion
		Strs("shards", rst.Client().Server().DatashardIds()[0]).
```

**Review by EinKrebs — CHANGES_REQUESTED** [review]

File: router/server/shard.go (hunk 2 above)
Comment (reshke) [reply router/server/shard.go:147]: thats because its now list of strings, not just one?

File: router/relay/relay.go (hunk 4 above)
Comment (Denchick) [reply router/relay/relay.go]: panic: runtime error: index out of range

File: router/server/shard.go (hunk 2 above)
Comment (Denchick) [reply router/server/shard.go]: panic: runtime error: index out of range

File: router/server/server.go (hunk 1 above)
Comment (Denchick) [reply router/server/server.go]: Add util function

**Review by reshke — APPROVED** [review]

**Pull request merged**